dired-narrow.el

```diff
@@ -95,25 +95,29 @@ we should act on."
                  (function :tag "Use custom function."))
   :group 'dired-narrow)
 
-(defcustom dired-narrow-exit-when-1-left nil
+(defcustom dired-narrow-exit-when-one-left nil
   "If there is only one file left while narrowing,
-exit minibuffer and call dired-narrow-exit-action."
+exit minibuffer and call `dired-narrow-exit-action'."
+  :type '(boolean)
```

Comment:
I saw this in the link that you provided :)
I also have tested customization interface and it worked properly.
I'll change it.